polars/polars-time/src/chunkedarray/utf8/mod.rs

```diff
@@ -421,13 +427,15 @@ pub trait Utf8Methods: AsUtf8 {
 
                 let mut convert = |s: &str| {
                     DateTime::parse_from_str(s, &fmt).ok().map(|dt| {
-                        match tz {
-                            None => tz = Some(dt.timezone()),
-                            Some(tz_found) => {
-                                if tz_found != dt.timezone() {
-                                    return Err(PolarsError::ComputeError(
-                                        "Different timezones found during 'strptime' operation.".into(),
-                                    ));
+                        if !utc {
+                            match tz {
+                                None => tz = Some(dt.timezone()),
+                                Some(tz_found) => {
+                                    if tz_found != dt.timezone() {
+                                        return Err(PolarsError::ComputeError(
+                                            "Different timezones found during 'strptime' operation.".into(),
```

Comment:
This error might now hint to using `utc=True`

Reply:
good one, thanks!